### DISCUSSION TITLE
Uyuni installation

### DISCUSSION THREAD

**Discussion by Alixis95** [uyuni-project/uyuni#7221] [Q&A]
Hello experts
I'm not familiar with Uyuni. 

I installed a Uyuni server to patch SAP systems installed on the SLES 15. 

I have created a chanel (in Uyuni) and now I want to insert a repository.
In the Uyuni web interface under Software->Manage->Repositories.

What should I enter there? I need repositories for
-SLES for SAP 15.SP 4,
- SLES for SAP 15 SP3,
SLES for SAP 15 SP2

Where can I find these repositories ?
Thanks very much
Best regards

**Comment by agraul** [comment]
Add first your SCC credentials and then the respective products at https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/admin/setup-wizard.html
